nixos/modules/services/desktops/playerctld.nix

```diff
@@ -0,0 +1,30 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.playerctld;
+in
+{
+  options.services.playerctld = {
+    enable = mkEnableOption (lib.mdDoc "the playerctld daemon");
+
+    package = mkPackageOption pkgs "playerctl" { };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+
+    systemd.user.services.playerctld = {
+      description = "Playerctld daemon to track media player activity";
+      wantedBy = [ "multi-user.target" ];
+
+      serviceConfig = {
+        Type = "exec";
+        ExecStart = "${cfg.package}/bin/playerctld";
+      };
+    };
+  };
+
+  meta = { maintainers = with lib.maintainers; [ aacebedo ]; };
```

Comment:
Use `meta.maintainers` please.

I would also argue that `with` should also be avoided here, since there is only one maintainer. But I'm not 100% sure since `with` is used in most places anyway, though I doubt that it's with good reason.

Reply:
I let it in case other maintainers are added then